core/src/banking_stage.rs

```diff
@@ -601,34 +624,65 @@ impl BankingStage {
         });
 
         // Spawn the central scheduler thread
-        bank_thread_hdls.push(
-            Builder::new()
-                .name("solBnkTxSched".to_string())
-                .spawn(move || {
-                    let scheduler = PrioGraphScheduler::new(
-                        work_senders,
-                        finished_work_receiver,
-                        PrioGraphSchedulerConfig::default(),
-                    );
-                    let scheduler_controller = SchedulerController::new(
-                        decision_maker.clone(),
-                        receive_and_buffer,
-                        bank_forks,
-                        scheduler,
-                        worker_metrics,
-                        forwarder,
-                    );
-
-                    match scheduler_controller.run() {
-                        Ok(_) => {}
-                        Err(SchedulerError::DisconnectedRecvChannel(_)) => {}
-                        Err(SchedulerError::DisconnectedSendChannel(_)) => {
-                            warn!("Unexpected worker disconnect from scheduler")
+        if use_greedy_scheduler {
```

Comment:
Possible to dedup any of this code?

Reply:
100% possible, I've already done it!
Trent told me to avoid moving/refactors in this PR because we intend to backport.

Reply:
simple is good